packages/echo-base/src/utils/eventHub.ts

```diff
@@ -47,7 +47,7 @@ class EventHub implements EchoEventHub {
      * @return {*} Returns cleanup function for unsubscribing from the subscribed events {() => void}
      * @memberof EventHub
      */
-    subscribeMany<T>(keys: Array<string | EchoEvents>, handler: (payload: T) => void): () => void {
+    subscribeMany<T>(keys: Array<string | EchoEvents>, handler: (payload: T) => void): UnsubscribeFunction {
         const unsubscribeFunctions: Array<() => void> = keys.map((key) => this.subscribe(key, handler));
         return (): void => {
             unsubscribeFunctions.map((unsubscribe) => {
```

Comment:
Consider using "forEach" instead of "map" as its return value is not being used here.

Reply:
it's being used on line 53